requirements.txt

```diff
@@ -1,4 +1,6 @@
 requests==2.26.0
 retry==0.9.2
 base58==2.1.1
-protobuf==3.19.3
\ No newline at end of file
+protobuf==3.19.3
+grpcio==1.44.0
+grpcio-tools==1.44.0
```

Comment:
I don't think grpcio-tools is required for runtime but needed for generating the code, we don't have a dev requirements.txt

Reply:
Feel free to make dev-requirements

Reply:
added. Assuming only additional ones have to go in there just like test-requirements